### PR TITLE
Pintu lecture

### PR DESCRIPTION
**Pull request by pintu8328**

### CODE REVIEW

File: src/Pages/AdminSidePages/AdminLecturePage/index.tsx [hunk 1]
```diff
@@ -47,11 +56,14 @@ const AdminLecture = () => {
   });
   const selectWidth = useBreakpointValue({ base: "100%", md: "auto" });
 
+
   return (
     <div className="container">
       <Navbar />
       <Secondnav />
 
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLecturePage/index.tsx:65]: why these extra spaces ??

File: src/Pages/AdminSidePages/AdminLogin/index.css [hunk 2]
```diff
@@ -14,6 +14,10 @@ body{
   width: auto;
   height: auto;
  background-color:white;
+ border: 1px solid rgb(41, 40, 40);
+ border-radius: 10px;
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLogin/index.css:19]: extra spaces ???

File: src/Pages/AdminSidePages/AdminLogin/index.css [hunk 3]
```diff
@@ -14,6 +14,10 @@ body{
...
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLogin/index.css:21]: extra spaces ??

File: src/Pages/AdminSidePages/AdminLogin/index.css [hunk 4]
```diff
@@ -28,11 +32,18 @@ justify-content: center;
   
   height:35px;
   width:90px;
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLogin/index.css:35]: extra spaces ??

File: src/Pages/AdminSidePages/AdminLogin/index.css [hunk 5]
```diff
@@ -28,11 +32,18 @@ justify-content: center;
...
   min-width: 90px;
+
   color:white;
      border-radius: 10px;
 }
 
 .button:hover{
+
+  background-color: black;
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLogin/index.css:46]: why spaces in between

File: src/Pages/AdminSidePages/AdminLogin/index.tsx [hunk 6]
```diff
@@ -1,6 +1,10 @@
 import * as yup from "yup";
 import { useFormik } from "formik";
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLogin/index.tsx:3]: why spaces inbetween

File: src/Pages/AdminSidePages/AdminLogin/index.tsx [hunk 7]
```diff
@@ -126,13 +184,19 @@ export default function AdminLogin() {
             w={["full", "md"]}
             p="10px 20px 20px 30px"
             mx="auto"
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminLogin/index.tsx:187]: remove these unnecessary spaces

File: src/Pages/AdminSidePages/AdminSignup/index.tsx [hunk 8]
```diff
@@ -63,11 +92,39 @@ const validationSchema = yup.object().shape({
...
+
+export default function AdminSignup() {
+  const [signupState, setSignupState] = useState(initialValues);
+  const navigate = useNavigate();
+  const handleSuccess = () =>
+    setSignupState({ ...signupState, captchaMatched: true });
+  const handleFailure = () =>
+    setSignupState({ ...signupState, captchaMatched: false });
+
+
```
Comment (mangesh0712) [inline src/Pages/AdminSidePages/AdminSignup/index.tsx:119]: why these extra spaces, ???? same mistakes again and again, you only need to add space between methods and sections not randomly, its not easy to read

File: src/Pages/ResetPassword/index.css [hunk 9]
```diff
@@ -0,0 +1,39 @@
+
+
+body{
+    background-color: rgb(243,244,246);
+  
+  }
```
Comment (mangesh0712) [inline src/Pages/ResetPassword/index.css:6]: put spaces after call, not in-between

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+    Select,
+    useBreakpointValue,
+    Flex,
+    Button,
+  } from "@chakra-ui/react";
+
+const StudentLecture = () => {
+    const [filterValues, setFilterValues] = useState({
+        title: "",
+        category: "",
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:18]: where is the interface for this state ?

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+        user: "",
+        date: "",
+        week: "",
+        day: "",
+      });
+    
+      const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+        const { name, value } = event.target;
+        setFilterValues({ ...filterValues, [name]: value });
+        console.log(filterValues);
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:30]: dont push console.log in the main branch, they for debugging purpose.

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 12]
```diff
@@ -0,0 +1,152 @@
...
+            <Box>
+            <Box>Category</Box>
+            <Select
+              name="category"
+              onChange={handleChange}
+              width={selectWidth}
+              placeholder="choose"
+              
+            >
+              <option value="Coding">Coding</option>
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:80]: can you pla use loop here for option

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 13]
```diff
@@ -0,0 +1,152 @@
...
+            </Box>
+             <Box><Box>Type</Box>
+            <Select
+              name="Type"
+              width={selectWidth}
+              placeholder="Choose"
+              
+            >
+              <option value="section1">Section 1</option>
+              <option value="section2">Section 2</option>
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:93]: use loop

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+            <Select
+              name="type"
+              width={selectWidth}
+              placeholder="Choose"
+             
+            >
+              <option value="type1">Type 1</option>
+              <option value="type2">Type 2</option>
+              <option value="type3">Type 3</option>
+              <option value="type4">Type 4</option>
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:107]: use loop

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 15]
```diff
@@ -0,0 +1,152 @@
...
+            <Box><Box>Optional</Box>
+             <Select
+              name="user"
+              width={selectWidth}
+              placeholder="Choose"
+              
+            >
+              <option value="user1">User 1</option>
+              <option value="user2">User 2</option>
+              <option value="user3">User 3</option>
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:127]: use loop

File: src/Pages/StudentSidePages/StudentLecturePage/index.tsx [hunk 16]
```diff
@@ -0,0 +1,152 @@
...
+            </Select>
+            </Box>
+          </Grid>
+           
+           
+
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLecturePage/index.tsx:133]: remove unnecessary comment

File: src/Pages/StudentSidePages/StudentLogin/Showerror.tsx [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+import "../../../App.css";
+interface ShowErrorProps {
+    children: React.ReactNode;
+  }
+
+const Showerror = (props :ShowErrorProps) => {
+  return (
+    <div>
+    <p className='email-error-showing-popup'>{props.children}</p>
+
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLogin/Showerror.tsx:11]: remove unnecessary spaces

File: src/Pages/StudentSidePages/StudentLogin/Showerror.tsx [hunk 18]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import "../../../App.css";
+interface ShowErrorProps {
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLogin/Showerror.tsx:3]: add space here

File: src/Pages/StudentSidePages/StudentLogin/index.css [hunk 19]
```diff
@@ -1,6 +1,43 @@
 
 
 body{
+
+    background-color: rgb(243,244,246);
+
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLogin/index.css:6]: remove spaces

File: src/Pages/StudentSidePages/StudentLogin/index.css [hunk 20]
```diff
@@ -1,6 +1,43 @@
...
 body{
+
+    background-color: rgb(243,244,246);
+  
+  }
+  .container{
+    height: auto;
+    
+  }
+
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentLogin/index.css:12]: remove spaces

File: src/Pages/StudentSidePages/StudentSignup/index.tsx [hunk 21]
```diff
@@ -128,6 +179,14 @@ const [isLoading,setLoading] = useState(false)
   };
 
 
+  // getBatchArrray().then((res)=>{
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentSignup/index.tsx:182]: remove commented code

File: src/Pages/StudentSidePages/StudentSignup/index.tsx [hunk 22]
```diff
@@ -146,13 +205,18 @@ const [isLoading,setLoading] = useState(false)
...
+
             h="auto"
+
             border={["none"]}
             bg="white"
             borderColor={["", "grey.300"]}
             borderRadius={10}
             boxShadow="2px 4px 6px rgba(0, 0, 0, 0.1)"
           >
+
```
Comment (mangesh0712) [inline src/Pages/StudentSidePages/StudentSignup/index.tsx:218]: remove unnecessary spaces

File: src/components/StudentSideComponents/StudentNavbar/Navbar.tsx [hunk 23]
```diff
@@ -0,0 +1,160 @@
...
+                  minH={"30px"}
+                  minW="80px"
+                  src={masaiimage}
+                  alt="Masai logo"
+                />
+              </Link>
+
+              <Hide below="md">
+                <Flex w="65%" justifyContent={"space-around"} align="center">
+                  <Box className="li">
```
Comment (mangesh0712) [inline src/components/StudentSideComponents/StudentNavbar/Navbar.tsx:53]: try to make common component box and loop trough the list of routes, by making array of const for routes

File: src/components/StudentSideComponents/StudentNavbar/Navbar.tsx [hunk 24]
```diff
@@ -0,0 +1,160 @@
...
+
+        {showProfile && <Profilecomponent setshow1={setShowProfile} />}
+
+        {/* this is part is like when user enter into smaller screens the navbar appear on left  side as side bar  */}
+
+        <div
+          id="navbar-mobile"
+          className={clicked ? "navbar-mobile active" : "navbar-mobile"}
+        >
+          <li>
```
Comment (mangesh0712) [inline src/components/StudentSideComponents/StudentNavbar/Navbar.tsx:114]: same for this li as well

File: src/Pages/ResetPassword/index.css (hunk 9 above)
Comment (sivagurubilli) [reply src/Pages/ResetPassword/index.css:6]: ok

**Pull request closed without merging**